### PR TITLE
Adding new Filters!

### PR DESCRIPTION
**Pull request by StephenP-Hung**
This PR represents the new filters that we're looking to add for SkunkBooth.

- Currently, implemented new filters for: 
  - Greek letters,
  - Musical notes 
  - Suits from a deck of cards

### CODE REVIEW

File: skunkbooth/filters/card_deck.py [hunk 1]
```diff
@@ -0,0 +1,33 @@
+from random import randint
+from typing import List, Tuple
+import random
+
+
+class filter:
+    """Sample filter for ASCII operations"""
```
Comment (johnperry99) [inline skunkbooth/filters/card_deck.py]: Maybe update all of these sample filter lines to be the name of the filter

File: skunkbooth/filters/greekify.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
+from random import randint
+import random
+from typing import List, Tuple
+
+
+class filter:
+    """Sample filter for ASCII operations"""
```
Comment (johnperry99) [inline skunkbooth/filters/greekify.py]: same as above comment

File: skunkbooth/filters/musical.py [hunk 3]
```diff
@@ -0,0 +1,33 @@
+from random import randint
+import random
+from typing import List, Tuple
+
+
+class filter:
+    """Sample filter for ASCII operations"""
```
Comment (johnperry99) [inline skunkbooth/filters/musical.py]: same as above comment

**Review by johnperry99 — APPROVED** [review]
LGTM!

File: skunkbooth/filters/card_deck.py (hunk 1 above)
Comment (StephenP-Hung) [reply skunkbooth/filters/card_deck.py]: Done!

File: skunkbooth/filters/greekify.py (hunk 2 above)
Comment (StephenP-Hung) [reply skunkbooth/filters/greekify.py]: Done!

File: skunkbooth/filters/musical.py (hunk 3 above)
Comment (StephenP-Hung) [reply skunkbooth/filters/musical.py]: And done!

**Pull request merged**